src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -134,6 +148,11 @@ export const authenticatorItem = ({
         ${nonNullish(dupCount) && dupCount > 0
           ? html`<i class="t-muted">&nbsp;(${dupCount})</i>`
           : undefined}
+        ${nonNullish(lastUsageFormattedString)
```

Comment:
I like the current text style but I would put them in two lines, what do you think?